content/introduction/introduction.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introduction"
+description: "Learn what SweetOps is and who it is for."
+icon: "fa fa-brain"
+---
+
+# What is SweetOps?
+
+SweetOps is a methodology for building modern, secure infrastructure on top of Amazon Web Services (AWS). It provides a toolset, library of reusable Infrastructure as Code (IaC), and opinionated patterns to help you bootstrap robust cloud native architectures. Built in an Open Source first fashion by [Cloud Posse](https://cloudposse.com/), it is utilized by many high performing startups to ensure their cloud infrastructure is an advantage instead of a liability. In short, SweetOps makes working in the DevOps world **Sweet**!
+
+# Who is this for?
+
+SweetOps is for DevOps or platform engineering teams that want an opinionated way to build software platforms in the cloud. If the below sounds like you, then SweetOps is what you're looking for:
+
+1. You're on AWS
```

Comment:
```suggestion
1. You're on AWS (while this is not a requirement, most of our terraform modules are design for AWS)
```

Reply:
I would suggest we not muddy the waters here... of course folks can use atmos, null-label, DD modules, etc without being on AWS, but our docs are going to be 100% AWS focused for the near future so if you're not on AWS then you're going to be lost quick. Thoughts?

Reply:
Ok, fair enough